packages/plugins/rubygem-foreman_ansible_core/rubygem-foreman_ansible_core.spec

```diff
@@ -18,8 +18,6 @@ Requires: %{?scl_prefix}rubygem(foreman-tasks-core) >= 0.1
 Requires: %{?scl_prefix}rubygem(foreman-tasks-core) < 1.0
 Requires: %{?scl_prefix}rubygem(foreman_remote_execution_core) >= 1.1
 Requires: %{?scl_prefix}rubygem(foreman_remote_execution_core) < 2.0
-Requires: %{?scl_prefix}rubygem(net-ssh) >= 4.0
```

Comment:
It shouldn't fully remove it, it's still a dependency. It's only the version constraint that has been removed.

Reply:
oh sure, so just keeping one line `Requires: %{?scl_prefix}rubygem(net-ssh)`

Reply:
Correct